packages/lib/src/components/BankTransfer/BankTransfer.tsx

```diff
@@ -50,16 +50,14 @@ export class BankTransferElement extends UIElement<BankTransferProps> {
             );
         }
 
-        if (this.props.showPayButton) {
-            return (
-                <CoreProvider i18n={this.props.i18n} loadingContext={this.props.loadingContext}>
-                    {this.props.showEmailAddress && (
-                        <BankTransferInput ref={this.handleRef} {...this.props} onChange={this.setState} />
-                    )}
+        return (
+            <CoreProvider i18n={this.props.i18n} loadingContext={this.props.loadingContext}>
+                {this.props.showEmailAddress && <BankTransferInput ref={this.handleRef} {...this.props} onChange={this.setState} />}
+                {this.props.showPayButton && (
                     <RedirectButton {...this.props} name={this.displayName} onSubmit={this.submit} payButton={this.payButton} />
-                </CoreProvider>
-            );
-        }
+                )}
+            </CoreProvider>
+        );
 
         return null;
```

Comment:
We might want to remove this one since it's unreachable (see Sonarcloud)

Reply:
sure!